modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,53 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+.Procedure
+
+. After creating the Argo CD instance, login into the keycloak console.
```

Comment:
Capitalize Keycloak Console since you capitalized `Keycloak Pod.` 

Be consistent with capitalizations throughout.